.github/workflows/pyopenms-wheels.yml

```
@@ -310,14 +307,8 @@ jobs:
     - name: Build on manylinux2014
       shell: bash
       run: |
+            sudo apt install -y jq
```

I think it is a CentOS image.
```suggestion
            sudo yum install -y jq
```